### PR TITLE
Calculate and return the wavelength where the max signal is achieved

### PR DESCRIPTION
**Pull request by cquiroz**
This is done in explore now but we could move this calculation to the backend

### CODE REVIEW

File: modules/itc/src/main/scala/lucuma/itc/Itc.scala [hunk 1]
```diff
@@ -90,6 +92,63 @@ object Itc:
     charts:      NonEmptyList[ItcChartGroup]
   )
 
+  object GraphResult:
+    def fromLegacy(
+      dataVersion: String,
+      ccds:        NonEmptyList[ItcRemoteCcd],
+      charts:      NonEmptyList[ItcChartGroup]
+    ): GraphResult = {
+      println(
```
Comment (rpiaggio) [inline modules/itc/src/main/scala/lucuma/itc/Itc.scala:101]: println

File: modules/itc/src/main/scala/lucuma/itc/Itc.scala [hunk 2]
```diff
@@ -90,6 +92,63 @@ object Itc:
...
+                         ccd.wellDepth,
+                         ccd.ampGain,
+                         ccd.warnings
+                  )
+                }
+              }
+              .toList
+              .flattenOption
+          }
+      assert(calculatedCCDs.length === ccds.length)
```
Comment (rpiaggio) [inline modules/itc/src/main/scala/lucuma/itc/Itc.scala:148]: Is the assert meant to be left in final code?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: modules/itc/src/main/scala/lucuma/itc/Itc.scala (hunk 2 above)
Comment (cquiroz) [reply modules/itc/src/main/scala/lucuma/itc/Itc.scala:140]: Yes I think so, this may alert us to logical errors if they ever happen

**Pull request merged**